tsconfig.json

```
@@ -11,5 +11,5 @@
     "declaration": true,
     "incremental": true
   },
-  "exclude": ["node_modules", "**/*.test.js"]
+  "exclude": ["node_modules", "**/*.test.js", "**/*.test.ts"]
```

Forgot about this feature.

```suggestion
  "exclude": ["node_modules", "**/*.test.js", "**/*.test.ts", "**/*.test.tsx"]
```